src/main/default/lwc/caseAssistFlow/__tests__/caseAssistFlow.test.js

```diff
@@ -545,6 +545,126 @@ describe('c-case-assist-flow', () => {
       // expect to send the event with coveoua
       expect(coveoua.mock.calls[2]).toEqual(EXPECTED_SEND_EVENT_PARAMS);
     });
+
+    it('should NOT send the TICKET_FIELD_UPDATE event with coveoua', async () => {
```

Comment:
This is the most verbose test i have ever seen :P
Is it really necessary to go through the full loop just to test that an event isn't sent? Could it be possible to mock some state and get straight to the punch?

my personal preference for tests is having as much of the variable declarations at the top of the test function (ex: expected events and param objects) that way it's easier to follow the steps of the test that come after.

Reply:
Well it is kinda necessary :) 
The first part is the boilerplate for the lwc...
After that it fills the subject and the description so the field section below appears with suggestions.
Then it clicks on a suggestion and checks that it was ONLY called with the ticket_classification_click.

It is very long, but at the same time, it is to test a very specific bug that I wouldn't want to have again.

I can move some of the expected stuff above yes.

Reply:
I trust you. I think I just went into it thinking this was a unit test but it really isn't